both/_collections.js

```diff
@@ -286,6 +287,22 @@ Registrations.attachSchema({
   },
   uniqueName: {
     type: String,
+    custom: function() {
+      var docId = this.docId;
+      var compId = this.field('competitionId').value;
+      var uniqueName = this.value;
+
+      if(Meteor.isServer) {
```

Comment:
Reading this code, I'm forgetting. The reason we check isServer is because docId is only defined on the server? If that's what's up, please add a quick comment and a link to https://github.com/aldeed/meteor-simple-schema/issues/208?